src/tfvc/tfvc.ts

```diff
@@ -139,24 +137,84 @@ export class Tfvc {
             TfvcOutput.AppendLine(`tf ${args.GetArgumentsForDisplay()}`);
         }
 
-        return cp.spawn(this._tfvcPath, args.GetArguments(), options);
+        return await this._tfRunner.Run(args, options, this._isExe);
     }
+}
 
-    private async _exec(args: IArgumentProvider, options: any = {}): Promise<IExecutionResult> {
-        const child = this.spawn(args, options);
+class TfRunner implements IDisposable {
+    private _location: string;
+    private _options: any;
+    private _runningInstance: cp.ChildProcess;
 
-        if (options.input) {
-            child.stdin.end(options.input, "utf8");
+    public constructor(location: string, options: any) {
+        this._location = location;
+        this._options = options;
+    }
+
+    public dispose() {
+        if (this._runningInstance) {
+            this._runningInstance.kill();
+            this._runningInstance = undefined;
         }
+    }
+
+    public async Run(args: IArgumentProvider, options: any, isExe: boolean): Promise<IExecutionResult> {
```

Comment:
It'd be great to have a comment about what's going on here.  After decoding it and recalling what you mentioned IntelliJ does, it makes more sense.  Something along the lines of "we're using the currently running instance to run the command but immediately start a new one so the next command can use it."